store_app/views/MovieCRUD.py

```diff
@@ -0,0 +1,65 @@
+from flask import make_response, jsonify, request
+from business import ConnectExternalAPI, MovieCRUD
+from app import app
+
+
+@app.route('/movies', methods=['POST', 'DELETE'])
```

Comment:
From what I read below this endpoint only deals with one movie at a time. Would it make sense to name this `/movie`?